test/runner.test.js

```diff
@@ -0,0 +1,38 @@
+const fs = require("fs");
+const path = require("path");
+const execa = require("execa");
+
+const TEMPLATES_DIR = path.resolve(__dirname, "..", "templates");
+
+const templates = fs.readdirSync(TEMPLATES_DIR);
+
+describe("npx create-snowpack-app", () => {
+  templates.forEach((template) => {
+    it(`--template @snowpack/${template}`, async () => {
+      // run yarn in this directory
+      await execa("yarn", { cwd: path.join(TEMPLATES_DIR, template) });
```

Comment:
Is there a way we could move this to the GitHub action itself? I see with Lerna you can run an npm script in every directory, but didn’t know if there was a way to run `yarn` or `npm install` in every package 🤔.

If we could move this easily to the GitHub Action itself, the test itself should be a lot safer/quicker

Reply:
Good as is IMO, but could be a good follow up enhancement down the road